extra-multimedia/sunshine/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=sunshine
+PKGDES="An open-source host-side implementation of NVIDIA's GameStream protocol"
```

Comment:
Please use the description from our CMakeLists file. https://github.com/LizardByte/Sunshine/blob/e1b112cafde0e1c668c7ae0d0f4592f5cdd0550a/CMakeLists.txt#L4